web/app/settings/settings.screensaver.html

```diff
@@ -0,0 +1,192 @@
+                                    </div>
+                                </div>
+                            </div>
+                        </li>
+                        <li>
+                            <label>
+                                <em translate-keep-content translate="screensaver.settings.slideshow.dashboardsExcluded.label">Excluded Dashboards:</em>
+                            </label>
+                            <div class="negative as-sortable" as-sortable="sortableOptions" ng-model="config.onStart.dashboardsExcluded">
+                                <div as-sortable-item ng-repeat="db in config.onStart.dashboardsExcluded | orderBy: db.order" class="d-block f-normal ss as-sortable-handle">
+                                    <div as-sortable-item-handle>
+                                        <span>{{db.id}} </span>
+                                        <span class="glyphicon glyphicon-move pull-right" title="{{::translations.reorder}}"></span>
+                                    </div>
+                                </div>
+                            </div>
+                        </li>
+                    </section>
+                    <section ng-if="config.onStart.type!=='slideshow'">
+                        <li>
+                            <label>
+                                <em translate-keep-content translate="screensaver.settings.common.dashboard">Dashboard:</em>
+                            </label>
+                            <select class="d-block" name='dashboard' ng-model="config.onStart.dashboard" ng-change="validate()" ng-init="config.onStart.dashboard = config.onStart.dashboard ? config.onStart.dashboard : dashboards[0].id">
+                                <option ng-repeat="db in dashboards | orderBy: db.id" value="{{db.id}}">{{db.id}}</option>
+                            </select>
+                        </li>
+                    </section>
+                </ul>
+            </div>
+            <div class="scrollable box">
+                <h4 translate="{{::translations.headingonstop}}"></h4>
+                <ul class="settings-outer">
+                    <li>
+                        <label translate-keep-content translate="screensaver.settings.common.perform" for="ss_onStopType">Perform: </label>
+                        <div>
+                            <select id="ss_onStopType" class="d-block" name='onStopType' ng-model="config.onStop.type" ng-change="validate()">
+                                <option translate-keep-content translate="screensaver.settings.common.stop" value='stop'>Stop</option>
+                                <option translate-keep-content translate="screensaver.settings.common.gotodashboard" value='gotodashboard'>Go to dashboard</option>
+                            </select>
+                            <span class='helpinfo'>
```

Comment:
Wonder whether this is really needed. Seems obvious, adds translation work and could be handled in the docs instead.